diff --git a/target/linux/ath79/dts/qca9531_glinet_gl-e750.dts b/target/linux/ath79/dts/qca9531_glinet_gl-e750.dts
--- a/target/linux/ath79/dts/qca9531_glinet_gl-e750.dts
+++ b/target/linux/ath79/dts/qca9531_glinet_gl-e750.dts
@@ -49,8 +49,8 @@
 	wifi@0,0 {
 		compatible = "qcom,ath10k";
 		reg = <0x0000 0 0 0 0>;
-		nvmem-cells = <&macaddr_art_0 2>;
-		nvmem-cell-names = "mac-address";
+		nvmem-cells = <&macaddr_art_0 2>, <&cal_art_5000>;
+		nvmem-cell-names = "mac-address", "calibration";
 	};
 };

@@ -105,6 +105,10 @@
 					cal_art_1000: calibration@1000 {
 						reg = <0x1000 0x440>;
 					};
+
+					cal_art_5000: calibration@5000 {
+						reg = <0x5000 0x844>;
+					};
 				};
 			};
