diff --git a/package/base-files/files/sbin/sysupgrade b/package/base-files/files/sbin/sysupgrade
--- a/package/base-files/files/sbin/sysupgrade
+++ b/package/base-files/files/sbin/sysupgrade
@@ -448,6 +448,8 @@ else
 	[ $USE_CURR_PART -eq 1 ] && json_add_boolean use_curr_part 1
 	json_close_object
 
+	[ -x /usr/libexec/mcu-notify ] && /usr/libexec/mcu-notify sysupgrade
+
 	ubus call system sysupgrade "$(json_dump)"
 fi
